Deck_Shuffler/Deck_Shuffler/Deck.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Deck_Shuffler
+{
+    class Deck
+    {
+        public List<Card> Cards = new List<Card>();
+        
+        public Deck()
+        {
+            string[] suits = new string[] { "Hearts", "Diamonds", "Clubs", "Spades" };
+            string[] values = new string[] { "Ace", "Two", "Three", "Four", "Five", "Six", "Seven", "Eight", "Nine", "Ten", "Jack", "Queen", "King" };
+            foreach (string suit in suits)
+            {
+                foreach (string value in values)
+                {
+                    Cards.Add(new Card(value, suit));
+                }
+            }
+        }
+        public Deck(string emptyDeck)
```

Comment:
What's this for?

Reply:
Making an empty deck - a deck with no cards in it (so that I can add cards to it later to make a full deck)

Reply:
OK. In that case I think it looks a bit weird as you just throw away emptyDeck. I'd be tempted to make two (static) methods on the Deck class `CreateOrderedDeck` and `CreateEmptyDeck`, which are called "factory methods". Both of these methods will use AddToDeck to build the appropriate deck. This would mean that Deck itself would no longer be doing anything in its constructor (the logic would have been moved to `CreateOrderedDeck`) so you could delete it.